assets/js/modules/analytics/dashboard/dashboard-widget-analytics-adsense-top-pages.js

```diff
@@ -146,24 +144,39 @@ const isDataZero = () => {
  *
  */
 const getDataError = ( data ) => {
-	if ( ! data || ! data.error ) {
-		return false;
-	}
+	if ( data.code && data.message && data.data && data.data.status ) {
+		// Specifically looking for string "badRequest"
+		if ( data.data.reason && 'badRequest' === data.data.reason ) {
+			return AnalyticsAdSenseDashboardWidgetTopPagesTable.renderLayout(
+				<CTA
+					title={ __( 'Restricted metric(s)', 'google-site-kit' ) }
+					description={ __( 'You need to link Analytics and AdSense to get report for your top earning pages. Learn more: https://support.google.com/adsense/answer/6084409 ', 'google-site-kit' ) }
```

Comment:
We should use `ctaLabel` and `ctaLink` props for "Learn more" and the URL rather than putting these in the description as plain text.